connector/src/main/java/org/geysermc/connector/network/translators/java/JavaChatTranslator.java

```diff
@@ -40,7 +41,17 @@ public void translate(ServerChatPacket packet, GeyserSession session) {
         TextPacket textPacket = new TextPacket();
         textPacket.setPlatformChatId("");
         textPacket.setSourceName("");
-        textPacket.setXuid(session.getAuthData().getXboxUUID());
+
+        // This attempts to find the XUID of the player so users can be muted/blocked
+        String xuid = "";
+        GeyserSession playerSession = GeyserConnector.getInstance().getPlayerByUuid(packet.getSenderUuid());
```

Comment:
Would it be better to ignore this check if the UUID most and least significant bits is 0? As that indicates it's a system message that should not be ignored, anyway.